presto-spi/src/main/java/com/facebook/presto/spi/statistics/Estimate.java

```diff
@@ -34,6 +34,11 @@ public static final Estimate unknownValue()
         return new Estimate(UNKNOWN_VALUE);
     }
 
+    public static final Estimate zeroValue()
```

Comment:
I have named it consistently with `unknownValue` named by Osipiuk on 28.03.17 :)